go/teams/handler.go

```diff
@@ -259,13 +259,27 @@ func HandleOpenTeamAccessRequest(ctx context.Context, g *libkb.GlobalContext, ms
 		var req keybase1.TeamChangeReq
 		joinAsRole := team.chain().inner.OpenTeamJoinAs
 
-		var needPost bool
+		var keybaseInvites []keybase1.UserVersion
+		var needPostMembership bool
 
 		for _, tar := range msg.Tars {
-			uv := NewUserVersion(tar.Uid, tar.EldestSeqno)
+			uv, err := loadUserVersionByUID(ctx, g, tar.Uid)
```

Comment:
I see no other way than to load user here. They will get loaded again during actual link posting (to generate recipient key boxes). Apart from server telling us which one should be membership and which one should be invite.